packages/react-core/src/components/DatePicker/examples/DatePicker.md

```diff
@@ -21,7 +21,11 @@ import React from 'react';
 import { DatePicker } from '@patternfly/react-core';
 
 AmericanFormat = () => {
-  const dateFormat = date => date.toLocaleDateString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit' });
+  const dateFormat = date => {
+    const localeDate = date.toLocaleDateString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit' });
+    const dasherizedDate = localeDate.replaceAll('/', '-');
+    return dasherizedDate;
+  };
   const dateParse = date => {
     const split = date.split('/');
```

Comment:
If you modify `dateFormat` you'll also have to modify `dateParse`.

BTW, why is our microcopy suggesting the American date format use `-` instead of `/` as the delimiter?

Reply:
Great catch! We recommended **MM-DD-YYYY** based on the existing microcopy guidelines provided for date and time formats on our PatternFly numerics page (https://www.patternfly.org/v4/ux-writing/numerics#date-and-time-formats). 

What do you think is the best solution? We could change the `-` to `/` format here and update the Numerics page to match, but we'd need to make sure we carry that through to Red Hat products that have existing date formats written as **MM-DD-YYYY**.

Curious to hear your thoughts on this too, @abigaeljamie!

Reply:
Hm, I think the PatternFly numerics page should reflect the international choice of `YYYY-MM-DD` that was signed off by design for the default DatePicker implementation. That's what [ISO8601](https://www.iso.org/iso-8601-date-and-time-format.html) and the [ECMA standard](https://262.ecma-international.org/11.0/#sec-date.parse) use.

Reply:
If design signed off on the international formatting (YYY-MM-DD) for the DatePicker implementation, I'm fine with changing the Numerics page to reflect that guidance. Would it be possible to change the component to reflect that, @evwilkin ?

Reply:
@abigaeljamie @afialkoff @redallen I'm going to pull these date formatting changes out of this PR for now, it sounds like a bigger discussion on not just how to proceed with this microcopy but with the guidelines for users and how changing this guidance will affect our users.  Created https://github.com/patternfly/patternfly-react/issues/6199 as a follow-up.